ksql-engine/src/test/java/io/confluent/ksql/integration/StreamsSelectAndProjectIntTest.java

```diff
@@ -66,80 +72,106 @@ public void after() throws Exception {
-        .DataSourceSerDe.AVRO, avroRecordMetadataMap);
+    testSelectProjectKeyTimestamp("PROJECT_KEY_TIMESTAMP_AVRO",
+                                  avroStreamName,
+                                  DataSource.DataSourceSerDe.AVRO,
+                                  avroRecordMetadataMap);
   }
 
   @Test
   public void testSelectProjectJson() throws Exception {
-    testSelectProject("PROJECT_STREAM_JSON", jsonStreamName, DataSource.DataSourceSerDe.JSON);
+    testSelectProject("PROJECT_STREAM_JSON",
+                      jsonStreamName,
+                      DataSource.DataSourceSerDe.JSON);
   }
 
   @Test
   public void testSelectProjectAvro() throws Exception {
-    testSelectProject("PROJECT_STREAM_AVRO", avroStreamName, DataSource.DataSourceSerDe.AVRO);
+    testSelectProject("PROJECT_STREAM_AVRO",
+                      avroStreamName,
+                      DataSource.DataSourceSerDe.AVRO);
   }
 
   @Test
   public void testSelectStarJson() throws Exception {
-    testSelectStar("EASYORDERS_JSON", jsonStreamName, DataSource.DataSourceSerDe.JSON);
+    testSelectStar("EASYORDERS_JSON",
+                   jsonStreamName,
+                   DataSource.DataSourceSerDe.JSON);
   }
 
   @Test
   public void testSelectStarAvro() throws Exception {
-    testSelectStar("EASYORDERS_AVRO", avroStreamName, DataSource.DataSourceSerDe.AVRO);
+    testSelectStar("EASYORDERS_AVRO",
+                   avroStreamName,
+                   DataSource.DataSourceSerDe.AVRO);
   }
 
 
   @Test
   public void testSelectWithFilterJson() throws Exception {
-    testSelectWithFilter("bigorders_json".toUpperCase(), jsonStreamName, DataSource.DataSourceSerDe.JSON);
+    testSelectWithFilter("BIGORDERS_JSON",
+                         jsonStreamName,
+                         DataSource.DataSourceSerDe.JSON);
   }
 
   @Test
   public void testSelectWithFilterAvro() throws Exception {
-    testSelectWithFilter("bigorders_avro".toUpperCase(), avroStreamName, DataSource.DataSourceSerDe.AVRO);
+    testSelectWithFilter("BIGORDERS_AVRO",
+                         avroStreamName,
+                         DataSource.DataSourceSerDe.AVRO);
   }
 
   @Test
   public void shouldSkipBadData() throws Exception {
     testHarness.createTopic(jsonTopicName);
     testHarness.produceRecord(jsonTopicName, "bad", "something that is not json");
-    testSelectWithFilter("bigorders_json1".toUpperCase(), jsonStreamName, DataSource.DataSourceSerDe
-        .JSON);
+    testSelectWithFilter("BIGORDERS_JSON1",
+                         jsonStreamName,
+                         DataSource.DataSourceSerDe.JSON);
   }
 
   @Test
   public void shouldSkipBadDataAvro() throws Exception {
     testHarness.createTopic(avroTopicName);
     testHarness.produceRecord(avroTopicName, "bad", "something that is not avro");
-    testSelectWithFilter("bigorders_avro1".toUpperCase(), avroStreamName, DataSource.DataSourceSerDe.AVRO);
+    testSelectWithFilter("BIGORDERS_AVRO1",
```

Comment:
Any particular reason to drop the `toUpperCase` method call here and elsewhere in this file?

Reply:
No reason, the value should be upper case and instead of calling uppercase method I just used the correct value.